### ISSUE TITLE
Add Tests

### ISSUE THREAD

**Issue by andrewrosemberg** [GiacomoBastianel/PowerModelsTopologicalActions#5]
For application packages, it is nice to have unit testing but, also, end-to-end tests. 

- [ ] End-to-end [PSCC_case_5.jl](https://github.com/GiacomoBastianel/PowerModelsTopologicalActions.jl/blob/main/PSCC_case_5.jl) is a great script to adapt to a test. If the others are fast as well, they can become tests.
- [ ] ... (EDIT THIS POST TO ADD LIST OF PACKAGE FUNCTIONS)

All of these should either live in a [`test`](https://github.com/lanl-ansi/PowerModels.jl/tree/master/test) folder. Which will contain a [runtests.jl](https://github.com/lanl-ansi/PowerModels.jl/blob/master/test/runtests.jl) script.

The end-to-end tests can live in an examples folder, from which files can be included in your `runtests.jl` script: [Examples-Tests](https://github.com/odow/SDDP.jl/blob/master/test/runtests.jl#L24-L27 )

**Comment by andrewrosemberg** [comment]
I imagine that the file you are using for the case 5 is the one [here](https://github.com/GiacomoBastianel/EU_grid_operations/blob/667c2a0dfc31668350c3937de92fe4407b4765e9/data_sources/case5_acdc.m) ?

**Comment by GiacomoBastianel** [comment]
Indeed, I added the three AC/DC test cases (5, 39 and 67 buses) again as they were deleted inadvertently
